diff --git a/node_modules/gatsby-plugin-prefetch-google-fonts/download-css.js b/node_modules/gatsby-plugin-prefetch-google-fonts/download-css.js
index b36d062..30b4531 100644
--- a/node_modules/gatsby-plugin-prefetch-google-fonts/download-css.js
+++ b/node_modules/gatsby-plugin-prefetch-google-fonts/download-css.js
@@ -30,7 +30,8 @@ function () {
               outputDir: path.join(".cache/google-fonts", config.pathPrefix),
               encode: false,
               minify: false,
-              verbose: true
+              verbose: true,
+              fontDisplay: _ref.fontDisplay
             });
             _context.next = 4;
             return googleFontsPlugin.make();
