diff --git a/node_modules/google-fonts-plugin/dist/index.js b/node_modules/google-fonts-plugin/dist/index.js
index e108ba9..cf2df7b 100755
--- a/node_modules/google-fonts-plugin/dist/index.js
+++ b/node_modules/google-fonts-plugin/dist/index.js
@@ -74,6 +74,10 @@ class GoogleFontsWebpackPlugin {
           });
         }
 
+        if (!this.options.encode && this.options.fontDisplay) {
+          requestString += '&display=' + this.options.fontDisplay;
+        }
+
         return requestString;
       }
 
@@ -188,7 +192,8 @@ _defineProperty(GoogleFontsWebpackPlugin, "defaultOptions", {
   },
   outputDir: 'public/fonts',
   encode: true,
-  minify: true
+  minify: true,
+  fontDisplay: 'swap'
 });
 
 module.exports = GoogleFontsWebpackPlugin;
