diff --git a/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/shared.js b/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/shared.js
index 92dbc0d..39e3e1f 100644
--- a/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/shared.js
+++ b/node_modules/react-cosmos/dist/plugins/webpack/webpackConfig/shared.js
@@ -12,15 +12,16 @@ var module_1 = require("./module");
 function getBaseWebpackConfig(cosmosConfig, userWebpack) {
     var rootDir = cosmosConfig.rootDir;
     var configPath = webpack_1.createWebpackCosmosConfig(cosmosConfig).configPath;
+    var defaultConfig = default_1.getDefaultWebpackConfig(userWebpack, rootDir);
     if (!configPath || !fs_1.moduleExists(configPath)) {
         console.log('[Cosmos] Using default webpack config');
-        return default_1.getDefaultWebpackConfig(userWebpack, rootDir);
+        return defaultConfig;
     }
     var relPath = path_1.default.relative(process.cwd(), configPath);
     console.log("[Cosmos] Using webpack config found at " + relPath);
     var userConfig = module_1.getDefaultExport(fs_1.requireModule(configPath));
     return typeof userConfig === 'function'
-        ? userConfig(process.env.NODE_ENV, yargs_1.argv)
+        ? userConfig(defaultConfig, { env: process.env.NODE_ENV, argv: yargs_1.argv })
         : userConfig;
 }
 exports.getBaseWebpackConfig = getBaseWebpackConfig;
